### DISCUSSION TITLE
Long term support of this addon?

### DISCUSSION THREAD

**Discussion by rhatguy** [matt-FFFFFF/hassio-addon-repository#16] [General]
Hi, this is somewhat of an interesting question.  I have an existing TeslaMate install running in a separate VM.  I also have a HomeAssistant VM that is regularly backed up through normal procedures.  Right now Teslamate and Home Assistant are only connected via MQTT.  I'd be interested in running your hassio addon, and take advantage of the synergies like leveraging my existing Hass.io backup process and ingress.  My concern is what would happen if for instance the HASS addon stopped being updated?  I have a lot of historical data in teslamate and intend to continue collecting data for years, so before I transition from my existing separate VM approach, I'd like to understand the long term support prospects for this addon.  Thanks!

**Comment by matt-FFFFFF** [comment]
Hi, thanks for your question.

As with most small/community projects there is no official support, nor any formal commitment to guarantee updates in the future. I can say though, at the moment I have no plans to drop this project.

What I would say is this, firstly the addon's data is all in a postgesql database. This can be backed up and restored independently. Therefore, you could migrate your data into a HASS hosted postgres and then export it if the need ever arose.

Secondly this maintenance of this addon does not take much of my time. It is trivial to update to a new upstream version of TeslaMate.

Thirdly, as this is completely open source, it is fully possible for somebody else to take over.

I hope this helps clarify the situation.
